src/utils/constants.js

```diff
@@ -31,7 +31,7 @@ export const initialCards = [
 
```

Comment:
* Please, style the `close icon` of the **image modal** correctly https://snipboard.io/GojQUL.jpg